lxd/instance/drivers/driver_qemu.go

```diff
@@ -1040,8 +1040,9 @@ func (d *qemu) saveState(monitor *qmp.Monitor) error {
 	return nil
 }
 
-// validateRootDiskStatefulStop validates the state of the root disk before stopping the instance.
-func (d *qemu) validateRootDiskStatefulStop() error {
```

Comment:
Why did you rename this, rather than only calling it during a stateful stop?

Now its being called during start up and not doing anything?

Reply:
It still need to check if the root disk device exists, that's why I kept it

Reply:
Lets just do that separately rather then bundle them together.